modules/default/compliments/compliments.js

```diff
@@ -55,7 +55,7 @@ Module.register("compliments", {
 
 		var self = this;
 		if (this.config.remoteFile != null) {
-			this.complimentFile((response) => {
+			this.complimentFile(function(response) {
 				this.config.compliments = JSON.parse(response);
```

Comment:
Just to let you know, this is not a correct port to es5. In order to work the same as the current version, you also have to replace this with self in this line, otherwise you cannot use the external file

Reply:
Thanks! I closed the pull request when the build failed and I realized it probably wasn't working properly, I just wasn't using a compliment file so it didn't show up locally.